DemoServer/Controllers/Demos/StaticIndexes/AdditionalSourcesIndex/metadata.json

```diff
@@ -0,0 +1,197 @@
+{
+    "Slug": "additional-sources-index",
+    "SourceFileName": "AdditionalSourcesIndexController.cs",
+    "Title": "Additional Sources Index",
+    "DescriptionLines": [
+        "Using the **Additional Sources** feature allows the index's Map & Reduce methods to reference code from external source files.",
```

Comment:
consider expand "reference" into "reference and use" since usage is what is actually crucial here

Reply:
Also, do we need to mention that we accept C# and JS as languages for Additional Sources?

Reply:
done